src/bika/lims/content/abstractroutineanalysis.py

```diff
@@ -510,3 +519,14 @@ def to_condition(condition):
         # Sanitize the conditions
         conditions = filter(None, [to_condition(cond) for cond in conditions])
         sample.setServiceConditions(other_conditions + conditions)
+
+    @security.public
+    def getPrice(self):
```

Comment:
I've moved the `getPrice` from `AbstractAnalysis` to `AbstractRoutineAnalysis`, because only analyses that inherit from the latter have a client associated